include/isaac/driver/buffer.h

```diff
@@ -54,8 +54,13 @@ class ISAACAPI Buffer: public has_handle_comparators<Buffer>
 
 public:
   //Constructors
+#ifdef _MSC_VER
+  Buffer(CUdeviceptr h, bool take_ownership = true);
```

Comment:
You forgot to initialize h to nullptr. And nullptr is supported by c++11, so please change to use nullptr for both windows and linux.